WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+
+import { Button } from 'semantic-ui-react';
+import I18n from '../../lib/i18n';
+import UserBadge from '../UserBadge';
+
+import '../../stylesheets/delegates/style.scss';
+
+const dasherize = (string) => string.replace(/_/g, '-');
+
+export default function Delegates({
+  delegates,
+  isEditVisible,
+}) {
+  const seniorDelegates = React.useMemo(() => delegates
+    .filter((user) => user.delegate_status === 'senior_delegate')
+    .sort((user1, user2) => {
+      if (user1.region < user2.region) {
+        return -1;
+      } if (user1.region > user2.region) {
+        return 1;
+      }
+      return 0;
+    }), [delegates]);
+
+  // TO_VERIFY: I assume there are no cases where there are no delegates without
+  // senior delegate unless they are senior delegate
+
+  return seniorDelegates.map((seniorDelegate) => {
+    const delegatesUnderSenior = [seniorDelegate, ...delegates
+      .filter((user) => user.senior_delegate_id === seniorDelegate.id && user.delegate_status !== 'trainee_delegate')
+      .sort((user1, user2) => {
+        if (user1.region < user2.region) {
+          return -1;
+        } if (user1.region > user2.region) {
+          return 1;
+        } if (user1.name < user2.name) {
+          return -1;
+        } if (user1.name > user2.name) {
+          return 1;
+        }
+        return 0;
+      })];
+    return (
+      <div
+        className="table-responsive"
+        key={`region-${seniorDelegate.id}`}
+      >
+        <table className="table delegates-table">
```

Comment:
Semantic UI React has dedicated tables: https://react.semantic-ui.com/collections/table/

Reply:
Thank you, migrated to semantic-ui table.